testcases/kdump/runkdump.sh

```diff
@@ -116,7 +125,11 @@ PrepareVerify ()
         COREDIR=/mnt"${COREDIR}"
     fi
 
+    if [ -x "/etc/init.d/kdump-tools" ]; then 
```

Comment:
I doubt that existence of this file implicates where core file is stored to.

Isn't the path saved somewhere in configuration file in /etc/ ?